### ISSUE TITLE
Move counter in pgn starts at 0

### ISSUE THREAD

**Issue by tryingsomestuff** [lucasart/c-chess-cli#38]
Isn't move counter shall start at 1 ?

```
[Round "1.4"]
[White "3.04_NiNy"]
[Black "3.04_NoNo"]
[Result "0-1"]
[Termination "checkmate"]
[FEN "rnbq1rk1/pp1pbpp1/4pn1p/6B1/2P5/P1N2N2/1PQ1PPPP/R3KB1R w KQ - 0 0"]
[PlyCount "134"]

0. Bh4 {38/21 36165ms} d6 {-24/20 32597ms} 1. Rd1 {22/21 28410ms} a6 {-34/22 20197ms}
```

I see lichess won't import the game if it starts at move named "0".

**Comment by lucasart** [comment]
c-chess-cli correctly deduces the move count from the given FEN, which is wrong. "0 0" is not allowed.

**Comment by lucasart** [comment]
fixed 4ae3db822f28989b902ef6b118fb28b3cd477a02.
